modules/seedtagBidAdapter.js

```diff
@@ -6,12 +6,7 @@ const BIDDER_CODE = "seedtag";
 const SEEDTAG_ALIAS = "st";
 const SEEDTAG_SSP_ENDPOINT = "https://s.seedtag.com/c/hb/bid";
 const SEEDTAG_SSP_ONTIMEOUT_ENDPOINT = "https://s.seedtag.com/se/hb/timeout";
-const ALLOWED_DISPLAY_PLACEMENTS = {
-  inImage: true,
-  inScreen: true,
-  inArticle: true,
-  inBanner: true,
-};
+const displayPlacements = ["inScreen", "inImage", "inArticle", "inBanner"];
```

Comment:
I liked the ALLOWED_DISPLAY_PLACEMENTS object and we could have another ALLOWED_VIDEO_PLACEMENTS.

But up to you this is fine and works!

Reply:
i didn't put another one for video, because the problem is it depends on the context. i could have put `["inStream"]` , but in reality i depends on the the context `instream`, that's why i didn't expose anything.
regarding the `ALLOWED_DISPLAY_PLACEMENTS` i found better to have an array of available placement as a string then an object where we check the key